src/main/java/io/aiven/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -324,12 +367,47 @@ public JdbcSinkConfig(final Map<?, ?> props) {
         timeZone = TimeZone.getTimeZone(ZoneId.of(dbTimeZone));
     }
 
-    private String getPasswordValue(final String key) {
-        final Password password = getPassword(key);
-        if (password != null) {
-            return password.value();
+    static Map<String, String> topicToTableMapping(final String value) {
+        return (Objects.nonNull(value))
+                ? Arrays.stream(value.split(","))
+                    .map(s -> s.split(":"))
+                    .collect(Collectors.toMap(e -> e[0], e -> e[1]))
+                : Collections.emptyMap();
+    }
+
+
+    public String generateTableNameFor(final String topic) {
+        final String tableName;
+        if (!topicsToTablesMapping.isEmpty()) {
+            tableName = topicsToTablesMapping.getOrDefault(topic, "");
+            if (tableName.isEmpty()) {
+                throw new ConnectException(String.format(
+                        "Destination table name for topic '%s'. Expected topics are: '%s'",
+                        topic,
+                        topicsToTablesMapping
+                                .entrySet()
+                                .stream()
+                                .map(e -> String.join("->", e.getKey(), e.getValue()))
+                                .collect(Collectors.joining(","))
+                ));
+            }
```

Comment:
would it be possible to fall back to the table name format generation if the table is not in the topic to table mapping?  this would provide behavior similar to the Snowflake connector

Reply:
Yes it is possible, will add test and implementation